config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```

Comment:
this file will not be up to GIt, please
```
cp database.yml database.yml.example
git rm --cached config/database.yml
```

and in .gitignore add
```
/config/database.yml
```
after that, you
```
git add .
git commit --amend --no-edit
git push your_branch -f
```

Reply:
fix it, please

Reply:
Done. I hope i've done it correctly